parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh

```diff
@@ -74,8 +74,14 @@ installNvidiaFabricManager() {
 }
 
 installNvidiaContainerToolkit() {
-    MARINER_NVIDIA_CONTAINER_TOOLKIT_VERSION="1.16.2"
-    
+    MARINER_NVIDIA_CONTAINER_TOOLKIT_VERSION=$(jq -r '.Packages[] | select(.name == "nvidia-container-toolkit") | .downloadURIs.azurelinux.current.versionsV2[0].latestVersion' /opt/azure/components.json)
```

Comment:
can we use this: https://github.com/Azure/AgentBaker/blob/6d639627bc8e85ab777f0ae2b121ceede4665925/parts/linux/cloud-init/artifacts/cse_install.sh#L26

instead of hardcoding `/opt/azure/components.json`?

Reply:
switched to the var :)